aea/crypto/base.py

```diff
@@ -224,3 +224,18 @@ def generate_tx_nonce(self, seller: Address, client: Address) -> str:
         :param client: the address of the client.
         :return: return the hash in hex.
         """
+
+
+class FaucetApi(ABC):
```

Comment:
@marcofavorito now we have a common interface here too

Reply:
shouldn't this be incorporated into the Ledger APIs? Hence, be part of a connection?

Reply:
Not really, a ledger api and a faucet are different. Faucets only exist for test nets and are usually not part of third party ledger apis